core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -86,7 +86,13 @@ class BasicWorkflow extends EnsimeSpec
           //-----------------------------------------------------------------------------------------------
           // symbolAtPoint
           project ! SymbolAtPointReq(Left(fooFile), 128)
-          val symbolAtPointOpt: SymbolInfo = expectMsgType[SymbolInfo]
+          expectMsgType[SymbolInfo]
+
+          project ! SymbolAtPointReq(Left(fooFile), 68) // scala.Int symbol
+          val symInfo = expectMsgType[SymbolInfo]
+          symInfo.`type`.pos should matchPattern {
+            case Some(OffsetSourcePosition(_, _)) =>
+          }
```

Comment:
we're missing the rest of the test here... we still need to lead the `scala.Int` source code and then do something with it, e.g. a `SymbolDesignationsReq`